### DISCUSSION TITLE
게시글이 없는 경우 예외 처리 필요

### DISCUSSION THREAD

**Discussion by EP45** [pronist/hELLO#243] [Ideas]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: Windows 11 24H2
- 브라우저: Edge
- 버전: 126.0.2592.87
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): 4.8.1

**설명**
게시글이 없는 경우 '글이 없습니다' 등 메시지가 표시되지 않고 무한 로딩처럼 보임.

**발생 과정**
게시글 0개인 블로그에 스킨 적용 시 발생.

**스크린샷 및 링크**
![화면 캡처 2024-07-13 015846](https://github.com/user-attachments/assets/e05d2d03-4599-4318-bc4d-c8d7069c30e3)

**Comment by pronist** [comment]
@EP45 게시글이 없는 경우에 대한 처리는 의도적으로 저 형태로 나타냈습니다만, 사용자가 이를 인지할 때 불편함이 초래된다면 패치를 고려해볼 수 있겠습니다.

버그가 아니므로 디스커션으로 이전되었습니다.

**Reply by EP45** [reply]
조금 더 확인해보니 전체 글이 0개 / 검색 결과가 0개 / 카테고리에 0개 등
반환되는 게시글이 0개면 위 스크린샷처럼 표시되는 것으로 보입니다.

로딩이 끝났음을 인지하게 하고, 다음 액션을 취하도록 유도하는 요소가 있으면 사용자 경험 향상에 도움이 될 것 같습니다.

다른 곳에선 어떤지 조금 찾아봤습니다.
1. [YouTube](https://www.youtube.com/)에 처음 들어가면 스켈레톤 로딩 후 `검색하여 시작하기` 가 표시됩니다.
2. [Odyssey](https://www.tistory.com/skin/Odyssey)의 경우 `아직 작성하신 글이 없어요.` 가 표시되고, 아래 상황에 따라 추가적인 액션을 유도하는 문자열이 표시됩니다.
ex) 블로그 주인이 로그인한 상태: `내 블로그의 첫 시작이 될 오늘의 기록을 남겨보세요!`
ex) 검색 결과가 없음: `입력하신 단어의 철자가 정확한지 확인해 보세요.` 등
ex) 카테고리가 비어 있음: `선택하신 카테고리에 해당하는 글이 없습니다.` 등

검토해 주시면 감사하겠습니다.
예쁜 스킨 만들어주셔서 감사합니다.
